app.py

```diff
@@ -64,14 +64,15 @@ def set_nofollow(attrs, new=False):
 __ALLOWED_TAGS =['p', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'br', 'hr', 'pre']
 @app.template_filter('markdown')
 def markdown_filter(s):
-    raw = bleach.clean(markdown.markdown('' if not s else s), 
+    raw = bleach.clean(markdown.markdown('' if not s else s),
                     tags=bleach.ALLOWED_TAGS+__ALLOWED_TAGS)
-    raw = bleach.linkify(raw, callbacks=[set_nofollow])
+    # raw = bleach.linkify(raw, callbacks=[set_nofollow])
+    raw = bleach.linkify(raw)
```

Comment:
given that no search indexes should be indexing us, if this fixes it I'm all for it.